apps/wallet-mobile/src/Dashboard/Dashboard.tsx

```diff
@@ -239,45 +248,50 @@ const EpochInfo = () => {
   )
 }
 
-const Actions = (props: ViewProps) => <View {...props} style={styles.actions} />
-
 const messages = defineMessages({
   stakingCenterButton: {
     id: 'components.delegation.delegationnavigationbuttons.stakingCenterButton',
     defaultMessage: '!!!Go to Staking Center',
   },
 })
 
-const styles = StyleSheet.create({
-  root: {
-    flex: 1,
-  },
-  container: {
-    flexDirection: 'column',
-    flex: 1,
-  },
-  scrollView: {
-    flex: 1,
-    backgroundColor: '#fff',
-  },
-  contentContainer: {
-    paddingTop: 16,
-    paddingHorizontal: 16,
-  },
-  row: {
-    flex: 1,
-    paddingVertical: 12,
-  },
-  actions: {
-    flexDirection: 'row',
-    backgroundColor: '#fff',
-    padding: 16,
-    borderTopLeftRadius: 8,
-    borderTopRightRadius: 8,
-    elevation: 1,
-    shadowOpacity: 0.06,
-    shadowColor: 'black',
-    shadowRadius: 6,
-    shadowOffset: {width: 0, height: -8},
-  },
-})
+const useStyles = () => {
+  const {theme} = useTheme()
+  const {color} = theme
+
+  const styles = StyleSheet.create({
+    root: {
+      flex: 1,
+    },
+    container: {
+      flexDirection: 'column',
+      flex: 1,
+    },
+    scrollView: {
+      flex: 1,
+      backgroundColor: color.gray.min,
+    },
+    contentContainer: {
+      paddingTop: 16,
+      paddingHorizontal: 16,
+    },
+    row: {
+      flex: 1,
+      paddingVertical: 12,
+    },
+    actions: {
+      flexDirection: 'row',
+      backgroundColor: color.gray.min,
+      padding: 16,
+      borderTopLeftRadius: 8,
+      borderTopRightRadius: 8,
+      elevation: 1,
+      shadowOpacity: 0.06,
+      shadowColor: 'black',
```

Comment:
black static

Reply:
Yeah, missed that when converting the raw styles to useStyles.